app/controllers/v2/content_items_controller.rb

```diff
@@ -12,6 +14,18 @@ def index
       ).call
     end
 
+    def linkables
+      render json: Queries::GetContentCollection.new(
+        content_format: request.query_parameters.fetch(:format),
+        fields: %w(
+          title
+          content_id
+          links
+          publication_state
```

Comment:
If we want to use this endpoint for the tagging, we need at least `base_path` and `details` here.